integration_tests/tests/s3.rs

```diff
@@ -234,12 +235,12 @@ fn test_multipart_upload(client: &TestClient, bucket: &str, filename: &str) {
     let part_req1 = create_upload_part(vec!['a' as u8; 1024 * 1024 * 5], 1);
     let part_req2 = create_upload_part("foo".as_bytes().to_vec(), 2);
 
-    // upload 2 parts and note the etags generated for them
+    // upload the parts and note the etags generated for them
     let mut completed_parts = Vec::new();
-    for req in vec![part_req1, part_req2].into_iter() {
-        let part_number = req.part_number;
+    {
```

Comment:
Are the added curly braces around the uploading parts needed?

Reply:
nope, I just thought it helped separate the two uploads visually a bit. happy to remove if you don't like 'em

Reply:
👍 thanks for the explanation.